onnxmltools/utils/float16_converter.py

```diff
@@ -0,0 +1,118 @@
+import numpy as np
+from struct import unpack
+from ..convert.common.data_types import *
+from onnxmltools.utils import load_model, save_model
+
+
+def npfloat16_to_int(np_list):
+    """
+    Convert numpy float16 to python int.
+    :param np_list: numpy float16 list
+    :return int_list: python int list
+    """
+    return [int(bin(_.view('H'))[2:].zfill(16), 2) for _ in np_list]
+
+
+def float16_converter(onnx_file, new_onnx_file):
```

Comment:
float16_converter sounds like a class, not a function. Maybe call this function `compress_model_float_to_float16`? I am not good at naming so you probably can find a better name.